### DISCUSSION TITLE
Which boundary conditions will be used in operations?

### DISCUSSION THREAD

**Discussion by DavidHuber-NOAA** [ufs-community/ufs-srweather-app#673] [Q&A] [answered]
Is the plan to use the operational GFS for lateral boundary conditions for the SRW, even the 0-hour?  If so, will it be forecasts from the most recent cycle available?

**Comment by ShawnCebulaNOAA** [comment]
Good morning David!

I'll check on this with the AUS team to see if they know about this, if not I'll see if I can get some direction to get this answered for you.  Thanks!

Shawn Cebula, EPIC End User Support

**Comment by EdwardSnyder-NOAA** [comment, marked as answer]
Good morning David,

My apologies for the late response. GFS data is used as the 0-hour lateral boundary conditions for the SRW App with the GFS data coming from the previous 6-hour run. Please let me know if you have any additional questions!

Thanks!

Eddie Snyder, EPIC Advanced User Support (AUS) team

**Reply by DavidHuber-NOAA** [reply]
Thanks @EdwardSnyder-NOAA!
